node.go

```diff
@@ -205,6 +205,7 @@ func (n *node) Wait() error {
 	defer n.finishedMu.Unlock()
 	if !n.finished {
 		n.finished = true
+		// deadlocking here
```

Comment:
Still?

Reply:
No, thanks, forgot to remove the comment.